__tests__/load-pgn.test.ts

```diff
@@ -29,6 +29,21 @@ describe('Load PGN', () => {
       ],
       expect: true,
     },
+    // GitHub Issue #362 - Load PGN works with no moves
+    {
+      pgn: [
+        '[Event "UrTurn Game Jam"]',
+        '[Round "6"]',
+        '[White "Kevo1ution"]',
+        '[Black "Billy Bob Jr."]',
+        '', // extra newlineChar is expected because pgn() function adds it
```

Comment:
I left this here because there is a behavior with `pgn()` where it outputs the pgn headers and an extra `newlineChar`. I didn't want to change the behavior of the pgn() function in this PR - It's not clear whether we should or shouldn't change the `pgn()` fn, but either way that change should be in a separate PR.

Reply:
Agree with the separate PR.  Would it be simple enough to just make `.pgn()` eat any trailing `newlineChar`?